nupic/torch/models/sparse_cnn.py

```diff
@@ -124,7 +124,7 @@ def __init__(self,
                  linear_units=1000,
                  linear_percent_on=0.1,
                  linear_weight_sparsity=0.4,
-                 boost_strength=1.5,
+                 boost_strength=1.67,
```

Comment:
Was this change done to compensate for the update before the first epoch?

Reply:
Yes. Not that critical, but I wanted the learning part to be identical to before.